### DISCUSSION TITLE
MS is stopping by itself

### DISCUSSION THREAD

**Discussion by rmusz** [Ylianst/MeshCentral#5468] [Issues]
Hi,

two maybe three times a day MC stops working on my server(ubuntu).

when its stopping to work my connection via putty to server shows message as in attached file

**Comment by rmusz** [comment]
![mc_error](https://github.com/Ylianst/MeshCentral/assets/148334921/db1c115f-8c5d-4cea-a3b6-cb9b9b56d665)

**Comment by si458** [comment]
what does your config.json look like?
are you using cloudflare?
what nodejs version are you using?

also im aware if the `warning an error event has already been emitted on the socket` message
i have noticed it too recently, i just havnt been able to track it down yet, but it doesnt seem to effect anything

**Comment by rmusz** [comment]
here you go ,

MC ver 1.1.14
nodejs 18.18.2
ubuntu 22.04

no im not using cloudfare

```
{
    "$schema": "https://raw.githubusercontent.com/Ylianst/MeshCentral/master/meshcentral-config-schema.json",
    "__comment1__": "This is a simple configuration file, all values and sections that start with underscore () are ignored. Edit a section and remove the in front of the name. Refer to the user's guide for details.",
    "__comment2__": "See node_modules/meshcentral/sample-config-advanced.json for a more advanced example.",
    "settings": {
        "MongoDb": "mongodb://127.0.0.1:27017/meshcentral",
        "cert": "SERVERHERE",
        "WANonly": true,
        "_sessionKey": "MyReallySecretPassword1",
        "port": 443,
        "_aliasPort": 443,
        "redirPort": 80,
        "_redirAliasPort": 80,
        "relayPort": 453
    },
    "domains": {
        "": {
            "title": "MYTITLEHERE",
            "_title2": "Servername",
            "_minify": true,
            "_newAccounts": true,
            "_userNameIsEmail": true
        }
    },
    "letsencrypt": {
        "__comment__": "Requires NodeJS 8.x or better, Go to https://letsdebug.net/ first before trying Let's Encrypt.",
        "email": "my@email.com",
        "names": "SERVERHERE",
        "skipChallengeVerification": true,
        "production": true
    }
}
```

**Reply by si458** [reply]
right so ur config is very basic, nothing special, 
when you say its stops working, do you mean u cant connect to any computers? either via terminal or desktop?
have you tried refreshing the page when it happens and see if it works again?

**Reply by rmusz** [reply]
i cannot access mc page. So i think mc service on server is stopped
